pkg/backingstore/reconciler.go

```diff
@@ -546,6 +565,9 @@ func (r *Reconciler) ReconcileExternalConnection() error {
 		return nil
 	}
 
+	if r.AddExternalConnectionParams == nil {
```

Comment:
**[IMPORTANT]**
I'm trying to figure out why this is needed, can you explain?
Perhaps best as a code comment...

Reply:
If I remember correctly, I believe I added this to avoid a subsequent error on line 615.